### PR TITLE
Swift support

### LINKED ISSUE #5: Include apple/swift-protobuf-plugin
Add Apple's own Swift protoc plugin at https://github.com/apple/swift-protobuf-plugin

### PR DESCRIPTION
**Pull request by QuentinPerez**
import from https://github.com/QuentinPerez/docker-alpine-swift-protobuf

closes #5

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -42,4 +46,9 @@ RUN apk add --no-cache build-base curl automake autoconf libtool git go zlib-dev
     find /usr/lib -name "*.a" -delete -or -name "*.la" -delete && \
     apk add --no-cache libstdc++
 
+ADD ./ressources/importlibpart1.tar /usr/lib/protoc-gen-swift-lib
+ADD ./ressources/importlibpart2.tar /usr/lib/protoc-gen-swift-lib
+ADD ./ressources/ld-linux-x86-64.so.2 /lib64/
+ADD ./ressources/protoc-gen-swift /usr/bin
```
Comment (moul) [inline Dockerfile]: those 4 `ADD` commands should be packed in only one to save layers 🌍

Comment (QuentinPerez) [inline Dockerfile]: Done, Thank you @moul <3

**Comment by jbdalido** [comment]
Since we're at it, let's use COPY instead of ADD, and keep this one for tar.gz and remote files :)

**Comment by jbdalido** [comment]
missfire as everyone saw i did not read properly ! my bad :D

**Comment by steeve** [comment]
Actually, can you package the resources as a tgz that we upload as a release on github ?
That won't pollute the git repo.

**Comment by steeve** [comment]
Just upload it on https://github.com/QuentinPerez/docker-alpine-swift-protobuf releases

**Pull request merged**